heap.py

```diff
@@ -0,0 +1,68 @@
+from dynamic_array import DynamicArray
+class MinHeap:
+    def __init__(self):
+        self.array = DynamicArray(10)
+
+    def add(self, value):
+        self.array.append(value)
+        self.heapify()
+
+    def delete(self):
+        if self.array.is_empty():
+            return False
+        self.swap_elements(0, self.array.fetch_size()-1)
+        self.array.current_index -= 1
+        self.array.size -= 1
+        self.heapify()
+        return True
+
+    def heapify(self):
+        child_index = self.array.fetch_size() - 1
+
+        while child_index > 0:
+            parent_index = (child_index - 1) // 2
+            child_element = self.array.get(child_index)
+            parent_element = self.array.get(parent_index)
+
+            if child_element is not None and parent_element is not None and child_element < parent_element:
```

Comment:
This can be simplified as `if x is not None` is same as `if x`. Pls simplify

Reply:
Done